playbook/app/pb_kits/playbook/pb_dialog/docs/_dialog_full_height.md

```diff
@@ -0,0 +1,2 @@
+The full height dialog can be size Medium, Large and XLarge.
```

Comment:
Did you mean to say small, medium, and large?

Reply:
Yes, the text is incorrect. thanks!